### ISSUE TITLE
README: Versions, Installation

### ISSUE THREAD

**Issue by ElfSundae** [ElfSundae/AFNetworking#1]
### Versions

| Version | Branch |                  Description                   |
|---------|--------|------------------------------------------------|
| 4.x     | master | Unofficial v4 releases with the latest commits  |
| 3.x     | 3.x    | Unofficial v3 releases merging from v4 commits |

- Release notes: https://github.com/ElfSundae/AFNetworking/releases
- Comparing changes:
    + [AFNetworking@master...ElfSundae@master](https://github.com/AFNetworking/AFNetworking/compare/master...ElfSundae:master)
    + [AFNetworking@master...ElfSundae:3.x](https://github.com/AFNetworking/AFNetworking/compare/master...ElfSundae:3.x)

#### CocoaPods

Specify `source` in your `Podfile`:

```ruby
pod 'AFNetworking', :source => 'https://github.com/ElfSundae/CocoaPods-Specs.git'
```

or:

```diff
+ source 'https://github.com/ElfSundae/CocoaPods-Specs.git'
  source 'https://cdn.cocoapods.org/'

  target 'Example' do
      pod 'AFNetworking'
  end
```

#### Carthage

```
github "ElfSundae/AFNetworking"
```

**Comment by waldviertler** [comment]
hello!
Please can you look at:
[https://stackoverflow.com/questions/60607164/afnetworking-elfsundae-uiwebview-patch
](url)
Thank you!

**Comment by ElfSundae** [comment]
@waldviertler Replied https://stackoverflow.com/a/60632560/521946

**Comment by hyunwoo3008** [comment]
Thank you
Will you update this code periodically?

**Comment by ElfSundae** [comment]
@hyunwoo3008 Certainly, I am using AFNetworking in many projects and I am watching the original repository on GitHub.

**Comment by Kaspik** [comment]
Main project released version 4.0.0.

**Comment by ElfSundae** [comment]
@Kaspik Thank you!
@waldviertler @hyunwoo3008 We can use the official v4 now 🎉 https://github.com/AFNetworking/AFNetworking/releases

**Comment by ElfSundae** [comment]
Released `4.0.0.1`, `3.2.2`
